### DISCUSSION TITLE
23.11.01 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#52] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 우재님 코드 리뷰
    - 친구 관계(`Relationship`) 엔티티 생성
    - 친구 요청 기능 개발
- 내일은 어떤 일을 하나요?
    - 포스트맨을 사용할 수 없는 문제 고민..
    - 내가 친구 요청 보낸 친구 조회 기능
    - 나에게 친구 요청 보낸 친구 조회 기능

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
    - 플레이리스트와 노래 매핑 + 테스트 코드 작성
- 내일은 어떤 일을 하나요?
    - 노래 : 스포티파이 API와 연동해서 데이터 가져오기
    - User와 플레이리스트 매핑
    - 우재님 코드 리뷰

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 로그 출력 기능 구현(AOP 실패로 excutor에서 출력)
  - 테스트 수정(실패)
- 내일은 어떤 일을 하나요?
  - 리팩토링 할 부분 찾기..?
  - 테스트 더 알아보기

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 통합 테스트
* 내일은 어떤 일을 하나요?
  * PR 올리신 엔티티 매핑

**Reply by Nine-JH** [reply]
1. Postman Redirect 문제 (인증정보를 제대로 담지 못하는 것 같음.)
2. 비동기 코드 시 통합 테스트 (TimeOut 이외에도 다른 방법이 있는지?)
